### ISSUE TITLE
Issue with conditionals that are not on the first page

### ISSUE THREAD

**Issue by HalldorLyngmo** [umbraco/Umbraco.Forms.Issues#462]
After an upgrade from Forms 8.1.4 to 8.6.0 a couple of issues emerged with a Form. When using conditionals on the first page and the following pages, the Forms JS will return exceptions for the following pages and break the conditionals on them.

```
VM8797 umbracoforms.js:529 Uncaught TypeError: Cannot read property 'style' of null
    at handleCondition (VM8797 umbracoforms.js:529)
    at Object.UmbracoFormsConditions.self.run (VM8797 umbracoforms.js:535)
    at Object.UmbracoFormsConditions.self.watch (VM8797 umbracoforms.js:403)
    at init (VM8797 umbracoforms.js:60)
    at initCollection (VM8797 umbracoforms.js:40)
    at listen (VM8797 umbracoforms.js:32)
    at VM8797 umbracoforms.js:20
    at VM8797 umbracoforms.js:551
```

Reproduction
------------

### Bug summary
I have a form where I'm using a condition on the first page. If you check the first field it will show the second field.
Same thing on page nr. 2 in the form.
![image](https://user-images.githubusercontent.com/42831978/101025157-74aef180-3575-11eb-8a7c-e1ec4989848e.png)
![image](https://user-images.githubusercontent.com/42831978/101025207-8d1f0c00-3575-11eb-829a-c23c1d949486.png)

The form is rendered with insert macro. I tried it with and without a theme. Got the same results:

![image](https://user-images.githubusercontent.com/42831978/101025385-cb1c3000-3575-11eb-877a-680fc024734f.png)

### What happens
When you open the page with the form there will be exceptions in the browser console. It will be possible to use the condition on the first page, but when you go to the second page it will not toggle the second field. Following the JS exception, it looks like the JS is trying to find the fields on the second page as soon as you open the form. Since the form fields for the second page haven't been loaded in, the JS finds nothing and returns an exception when trying to set the display property.

![Forms-issue](https://user-images.githubusercontent.com/42831978/101026807-a45ef900-3577-11eb-96c6-c9cd77e25eb8.gif)

As you can see on the image here. The element is null because it can't be found
![image](https://user-images.githubusercontent.com/42831978/101027754-bb521b00-3578-11eb-85c4-6bf6455938a4.png)

### Specifics
Using Chrome - Version 86.0.4240.198 (Official Build) (64-bit)
Umbraco v8.9.1
Umbraco Forms v8.6.0

This was tested on Cloud and I haven't tested on older versions of Forms. I believe this might have been introduced with the latest changes to remove the JQuery dependencies 

## EDIT:
I've done another test using Forms v8.5.5, adding the JQuery as suggested here - https://our.umbraco.com/documentation/Add-ons/UmbracoForms/Developer/Prepping-Frontend/#use-jquery-required-before-86

The same setup as above with the conditionals on two pages and the functionality works fine on each page without any exceptions in the browser console. 

After upgrading to Forms v8.6.0 and changing nothing. Still using JQuery, we have the same issue with conditionals not working. I tried a couple of different setting options, such as excluding the scripts and adding them like this but it was still broken. https://our.umbraco.com/documentation/Add-ons/UmbracoForms/Developer/Rendering-Scripts/#rendering-forms-scripts-where-you-want

---
_This item has been added to our backlog [AB#9702](https://dev.azure.com/umbraco/243e7927-03b2-44e2-908f-d4ac7ea5daaa/_workitems/edit/9702)_

**Comment by JHHessels** [comment]
I had the same problem and this was the solution for me:

open App_Plugins\UmbracoForms\Assets\themes\default\umbracoforms.js

And replace the function handleCondition with

function handleCondition (element, id, condition, type) {
                var shouldShow = isVisible (id, condition);

                if (element) {
                    if (shouldShow) {
                        element.style.display = "";
                    }
                    else {
                        element.style.display = "none";
                    }
                }
            }
			
And don't forget https://our.umbraco.com/documentation/Add-ons/UmbracoForms/Developer/Prepping-Frontend/#preparing-your-frontend

**Comment by Jeavon** [comment]
Thanks, @JHHessels fixed a form we upgraded from 8.4 to 8.7rc

@AndyButland worth a look at for the 8.7 release?

**Comment by AndyButland** [comment]
Agreed @Jeavon - have tested myself, and can replicate the original problem and verified the fix.  Thanks very much @JHHessels for sharing.  Given there's been at least three verifications I'll include this 8.7 due next week.
